index.d.ts

```diff
@@ -0,0 +1,35 @@
+import {Request, Response, Handler} from "express"
+
+declare namespace ExpressMongoSanitize {
+    interface Options {
+        replaceWith?: string;
+        onSanitize?: (req: Request, res: Response) => unknown;
+        dryRun?: boolean;
+    }
+}
+
+
+type Middleware = {
+    /**
+     * Create middleware instance
+     * @param options
+     */
+    (options?: ExpressMongoSanitize.Options): Handler;
+    /**
+     * Remove any keys containing prohibited characters
+     * @param target
+     * @param options
+     */
+    sanitize<T extends object>(target: T, options?: ExpressMongoSanitize.Options): T;
```

Comment:
Should we use something other than object here? Maybe Record<string, unknown>? Or { [key: string]: unknown } ?

Also see: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md

Reply:
Thanks.
Change to `Record<string, unknown> | unknown[]`.